.github/workflows/setupgo118amd64/action.yaml

```diff
@@ -0,0 +1,14 @@
+name: 'Setup go and vendor'
+description: 'Setup go with our auth'
```

Comment:
Checkout livepatches updated version of this, ill eventually put it in a reusable one for us so we can share it

Reply:
done